<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>无人机回传数据</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTabWidget" name="uavStatusTabWidget">
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>1</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTabWidget" name="tabWidget_2">
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>2</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QTabWidget" name="tabWidget_3">
           <widget class="QWidget" name="tab_7">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_8">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>标记点</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="exportTagsBtn">
               <property name="text">
                <string>导出数据</string>
               </property>
               <property name="icon">
                <iconset resource="qavserver.qrc">
                 <normaloff>:/images/icos/export.png</normaloff>:/images/icos/export.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="importTagsBtn">
               <property name="text">
                <string>导入数据</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>images/icos/import.png</normaloff>images/icos/import.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tagXLineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tagYLineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>名称:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tagNameLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="addTagPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qavserver.qrc">
                 <normaloff>:/images/icos/add.ico</normaloff>:/images/icos/add.ico</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="delTagPushButton">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="qavserver.qrc">
                 <normaloff>:/images/icos/minus.ico</normaloff>:/images/icos/minus.ico</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tagTableWidget">
             <column>
              <property name="text">
               <string>名称</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>X坐标</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Y坐标</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="uavTypeDataGroup">
          <property name="title">
           <string>连接状态</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QListWidget" name="uavslistWidget">
               <property name="styleSheet">
                <string notr="true">#uavslistWidget {
	border: none;
	background-color: transparent;
}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="openServerBtn">
               <property name="text">
                <string>开启连接</string>
               </property>
               <property name="icon">
                <iconset resource="qavserver.qrc">
                 <normaloff>:/images/icos/disconnected.png</normaloff>:/images/icos/disconnected.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1">
             <item>
              <widget class="QLabel" name="uavImageLabel">
               <property name="text">
                <string>无人机图片预览</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="uavParameterPlainTextEdit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">#uavParameterPlainTextEdit {
	border: none;
	background-color: transparent;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="rOg_image" name="gisView"/>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>路径</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="importPathAirspaceBtn">
             <property name="text">
              <string>导入数据</string>
             </property>
             <property name="icon">
              <iconset resource="qavserver.qrc">
               <normaloff>:/images/icos/import.png</normaloff>:/images/icos/import.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportPathAirspaceBtn">
             <property name="text">
              <string>导出数据</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>images/icos/export.png</normaloff>images/icos/export.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>规划路径</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="pathXLineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="pathYLineEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="addPathPushButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="qavserver.qrc">
                   <normaloff>:/images/icos/add.ico</normaloff>:/images/icos/add.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delPathPushButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="qavserver.qrc">
                   <normaloff>:/images/icos/minus.ico</normaloff>:/images/icos/minus.ico</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="pathTableWidget">
               <column>
                <property name="text">
                 <string>X坐标</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Y坐标</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>可行空域</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="airspaceXLineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="airspaceYLineEdit"/>
               </item>
               <item>
                <widget class="QPushButton" name="addAirspaceBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="qavserver.qrc">
                   <normaloff>:/images/icos/add.ico</normaloff>:/images/icos/add.ico</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="delAirspaceBtn">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="qavserver.qrc">
                   <normaloff>:/images/icos/minus.ico</normaloff>:/images/icos/minus.ico</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="airspaceTableWidget">
               <column>
                <property name="text">
                 <string>X坐标</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Y坐标</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>实际飞行路径</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <widget class="QTableWidget" name="realPathTableWidget">
               <column>
                <property name="text">
                 <string>X坐标</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Y坐标</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openMap"/>
   <addaction name="uavParameter"/>
   <addaction name="weatherBtn"/>
  </widget>
  <action name="openMap">
   <property name="icon">
    <iconset resource="qavserver.qrc">
     <normaloff>:/images/icos/map48x48.ico</normaloff>:/images/icos/map48x48.ico</iconset>
   </property>
   <property name="text">
    <string>打开地图</string>
   </property>
  </action>
  <action name="uavParameter">
   <property name="icon">
    <iconset resource="qavserver.qrc">
     <normaloff>:/images/icos/uavBtn.ico</normaloff>:/images/icos/uavBtn.ico</iconset>
   </property>
   <property name="text">
    <string>无人机参数</string>
   </property>
   <property name="toolTip">
    <string>修改无人机参数</string>
   </property>
  </action>
  <action name="weatherBtn">
   <property name="icon">
    <iconset resource="qavserver.qrc">
     <normaloff>:/images/icos/weather.ico</normaloff>:/images/icos/weather.ico</iconset>
   </property>
   <property name="text">
    <string>天气要素</string>
   </property>
   <property name="toolTip">
    <string>更改天气要素</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>rOg_image</class>
   <extends>QGraphicsView</extends>
   <header>components/rog_image.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qavserver.qrc"/>
 </resources>
 <connections/>
</ui>
